components/not-found.jsx

```diff
@@ -9,9 +9,9 @@ var NotFoundMessage = function(props) {
                 height='auto'
                 src1x='/img/pages/not-found/book_singlepageflip.gif'
                 alt='' />
-      <h2>Hey, this 404 is a teachable moment!</h2>
-      <p>Did you know that a 404 is the generic error code used across the internet to mean "page not found"? It might mean the page used to exist, but doesn't anymore. Or that someone made a typo somewhere. Regardless, there's no page at this address.</p>
-      <Link to={"/" + this.context.intl.locale + "/"} className="btn">Go To Home Page</Link>
+      <h2>{this.context.formatMessage({id: 'hey_404'})}</h2>
```

Comment:
+1 to that (I was checking with Pomax if that seemed reasonable or not, he just beat me to the comment 😛 )